### PR TITLE
add gnomad column

### PR DESCRIPTION
**Pull request by leexgh**
Add Gnomad column showing the population, allele count, allele number, homozygotes and frequency in the pop-up table.

### Have gnomAD data - not 0
Column: total frequency
Tooltip: gnomAD table

image

### Have gnomAD data - All allele frequencies are 0
Column: 0
Tooltip: gnomAD table

image

### No gnomAD data
Column: blank
Tooltip: `Variant has no data in gnomAD`

image

### Table explanation
image

Other headers:
- Allele Count: Number of individuals with this allele
- Allele Number: Number of times any allele has been observed at this position in the population
- Number of Homozygotes: Number of individuals carrying this allele in both copies
- Allele Frequency: Proportion of the population with this allele

Examples:
https://cbioportal-frontend-pr-2064.herokuapp.com/results/mutations?Action=Submit&RPPA_SCORE_THRESHOLD=2.0&Z_SCORE_THRESHOLD=2.0&cancer_study_list=brca_sanger&case_set_id=brca_sanger_sequenced&data_priority=0&gene_list=TP53&geneset_list=%20&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=brca_sanger_mutations&tab_index=tab_visualize

https://cbioportal-frontend-pr-2064.herokuapp.com/results/mutations?Action=Submit&RPPA_SCORE_THRESHOLD=2.0&Z_SCORE_THRESHOLD=2.0&cancer_study_list=ov_tcga_pub&case_set_id=ov_tcga_pub_cna_seq&data_priority=0&gene_list=BRCA1&geneset_list=%20&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=ov_tcga_pub_gistic&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=ov_tcga_pub_mutations&tab_index=tab_visualize

### CODE REVIEW

**Comment by jjgao** [comment]
@leexgh could you add a screenshot here? I'd like to get some feedback from collaborators.

**Comment by leexgh** [comment]
@jjgao Sure, add one above

**Comment by jjgao** [comment]
@leexgh is there data of "numbers of homozygotes" from myvariantinfo?

**Comment by leexgh** [comment]
@jjgao yes

**Comment by jjgao** [comment]
@leexgh please change the header to `gnomAD`

**Comment by jjgao** [comment]
@leexgh Could you post the link to the example you are using? https://cbioportal-frontend-pr-2064.herokuapp.com

**Comment by leexgh** [comment]
@jjgao Sorry I can't reproduce on heroku now. The screenshot is taken on local cbioportal with local genome nexus, I'm waiting for myvariantInfo PR merging (https://github.com/genome-nexus/genome-nexus/pull/267) @onursumer @inodb. I will send you the link when everything is set up

**Comment by jjgao** [comment]
@leexgh got it. Thanks!

**Comment by leexgh** [comment]
@jjgao Here is an example:
https://cbioportal-frontend-pr-2064.herokuapp.com/results/mutations?Action=Submit&RPPA_SCORE_THRESHOLD=2.0&Z_SCORE_THRESHOLD=2.0&cancer_study_list=ov_tcga_pub&case_set_id=ov_tcga_pub_cna_seq&data_priority=0&gene_list=BRCA1&geneset_list=%20&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=ov_tcga_pub_gistic&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=ov_tcga_pub_mutations&tab_index=tab_visualize

File: end-to-end-tests/specs/genome-nexus/mutationTable.spec.js [hunk 1]
```diff
@@ -81,4 +81,46 @@ describe('Mutation Table', function() {
...
+            // find frequency
+            const frequency = '[data-test2="TCGA-78-7540-01"][data-test="gnomad-column"]';
+            browser.waitForExist(frequency, 30000);
+            // wait for gnomad frequency show in the column
+            browser.waitUntil(() => {
+                var textFrequency = browser.getText(frequency);
+                return textFrequency.length >= 1;
+            }, 600000, "Frequency data not in Gnoamd column");
+            // check if the column has <0.0001 
+            assert.equal(browser.getText(frequency), '<0.0001');
```
Comment (inodb) [inline end-to-end-tests/specs/genome-nexus/mutationTable.spec.js]: @leexgh this prolly needs to be updated (no longer showing that)

Comment (leexgh) [reply end-to-end-tests/specs/genome-nexus/mutationTable.spec.js]: @inodb sure will do

**Comment by leexgh** [comment]
fixed issue #6111 #6112 #6114

**Comment by leexgh** [comment]
fixed https://github.com/cBioPortal/cbioportal/issues/6113

**Comment by leexgh** [comment]
fixed https://github.com/cBioPortal/cbioportal/issues/6190

File: src/shared/cache/GenomeNexusMyVariantInfoCache.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import * as _ from "lodash";
```
Comment (inodb) [inline src/shared/cache/GenomeNexusMyVariantInfoCache.ts]: @leexgh there'a lot of duplication by making a copy of the cache file like this. There might be a way to instead change the constructor to add a parameter that indicates what fields to add. We can fix it at a later stage if we end up needing more caches

**Comment by inodb** [comment]
@leexgh Nice work 🎉 !

**Comment by inodb** [comment]
Failing e2e are unrelated to this PR:

https://github.com/cBioPortal/cbioportal-frontend/pull/2443

**Pull request merged**